packages.md

```diff
@@ -35,6 +35,26 @@ Or you can do it with [`LOOP`][loop].
   collect s)
 ~~~
 
+# Give package local nickname
+
+Sometimes it is handy to give imported a local name for saving some typing. 
```

Comment:
IMO:

sometimes it is handy to give a local name to an imported package to save some typing.

Reply:
To be honest these contents was finished before a meeting so I was really in hurry.

I apologize for not double-checking before committing.

Reply:
Fixed. Please check.